resources/options_lookup.tsv

```diff
@@ -9912,26 +9912,26 @@ Hot Water Distribution	"R-2, Timer"	ResStockArguments	hot_water_distribution_sys
-Hot Water Fixtures	100% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.0
-Hot Water Fixtures	110% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.1
-Hot Water Fixtures	120% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.2
-Hot Water Fixtures	130% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.3
-Hot Water Fixtures	140% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.4
-Hot Water Fixtures	150% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.5
-Hot Water Fixtures	160% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.6
-Hot Water Fixtures	170% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.7
-Hot Water Fixtures	180% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.8
-Hot Water Fixtures	190% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=1.9
-Hot Water Fixtures	200% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=2.0
-Hot Water Fixtures	40% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=0.4
-Hot Water Fixtures	50% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=0.5
-Hot Water Fixtures	60% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=0.6
-Hot Water Fixtures	70% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=0.7
-Hot Water Fixtures	80% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=0.8
-Hot Water Fixtures	90% Usage	ResStockArguments	water_fixtures_shower_low_flow=false	water_fixtures_sink_low_flow=false	water_fixtures_usage_multiplier=0.9
+Hot Water Fixtures	"100% Usage, Low Flow"	ResStockArguments	water_fixtures_shower_low_flow=true	water_fixtures_sink_low_flow=true	water_fixtures_usage_multiplier=1.0	general_water_use_usage_multiplier=auto
```

Comment:
It has very little to do with hot water, so this is an odd place to put it. How about assigning it next to number of occupants?

Reply:
Works for me.